crates/runtime-config/src/lib.rs

```diff
@@ -156,33 +156,6 @@ where
         })
     }
 
-    /// Set initial key-value pairs supplied in the CLI arguments in the default store.
-    pub async fn set_initial_key_values(
-        &self,
-        initial_key_values: impl IntoIterator<Item = &(String, String)>,
-    ) -> anyhow::Result<()> {
-        // We don't want to unnecessarily interact with the default store
-        let mut iter = initial_key_values.into_iter().peekable();
-        if iter.peek().is_none() {
-            return Ok(());
-        }
-
-        let store = self
-            .key_value_resolver
-            .default(DEFAULT_KEY_VALUE_STORE_LABEL)
-            .expect("trigger was misconfigured and lacks a default store")
-            .get(DEFAULT_KEY_VALUE_STORE_LABEL)
-            .await
-            .expect("trigger was misconfigured and lacks a default store");
-        for (key, value) in iter {
-            store
-                .set(key, value.as_bytes())
-                .await
-                .context("failed to set key-value pair")?;
-        }
-        Ok(())
-    }
```

Comment:
@lann @rylev I'm not 100% confident this can be removed. It compiles without it and I can't find references to it. But, still not super sure.

Reply:
I'm 90% sure this is a refactoring artifact.

Reply:
I'm 100% sure this is just an artifact 😄